### ISSUE TITLE
Loss function for masked face 

### ISSUE THREAD

**Issue by anti-machinee** [deepinsight/insightface#2018]
- I have researched some marginal softmax as a head of Face recognition model. There are variable architectures such as ArcFace, CosFace, AdaCos ... Most of the heads based on marginal and we need to choose some parameter s, m ...
- Are there any experiments that choose these parameters by training them?
- When facing to masked face recognition problem, should we add weights to the marginal softmax layers depending on class masked or no masked?
